examples/edge-stack-iso/kubernetes/helm/values/neuvector.yaml

```diff
@@ -0,0 +1,19 @@
+# Based on https://gist.github.com/agracey/6966338d34c7277f5b5ec6f4c1431b1b
```

Comment:
I'm using your example here @agracey is this right?

Reply:
Better use this https://github.com/suse-edge/charts/blob/main/charts/neuvector-core/2.6.6/values.yaml